src/Sentry.OpenTelemetry/SentrySpanProcessor.cs

```diff
@@ -113,7 +113,7 @@ public override void OnEnd(Activity data)
         // Make a dictionary of the attributes (aka "tags") for faster lookup when used throughout the processor.
         var attributes = data.TagObjects.ToDict();
 
-        if (attributes.TryGetTypedValue("http.url", out string? url) && (_options?.IsSentryRequest(url) ?? false))
+        if (attributes.TryGetTypedValue("url.full", out string? url) && (_options?.IsSentryRequest(url) ?? false))
```

Comment:
We should probably use the consts defined in:
https://github.com/getsentry/sentry-dotnet/blob/bf4e0b3db4df28447d055cb569093c4307e85cf7/src/Sentry/Internal/OpenTelemetry/SemanticConventions.cs#L128

As you can see from the comments, however, the key will depend on the version of OTel that was used to generate the span... and I'm not sure if we have any control over that. So we might need to check for both the old and the new value. 

I think it was changed in v1.5.1-beta.1 of OpenTelemetry and we take a dependency on 1.5.0 at the moment:
https://github.com/getsentry/sentry-dotnet/blob/dd9cfac482c7759f9742db495e2e3750967449f5/src/Sentry.OpenTelemetry/Sentry.OpenTelemetry.csproj#L18-L20